### PR TITLE
Add WorkspaceMember

### PR DESCRIPTION
**Pull request by topecongiro**
The implementation of WorkspaceMember is almost identical to PackageId in cargo.
The difference is that its deserialization is simplified and it does not use Arc internally.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
+cargo = "0.23"
```
Comment (oli-obk) [inline Cargo.toml]: I'd rather be independent from the cargo crate. Can we just use `String` instead of `SourceId` for now?

**Comment by topecongiro** [comment]
Thank you for the review, I replaces `SourceId` with `String`.

**Comment by oli-obk** [comment]
Ok, so this'll be a breaking change. I'll check whether there are other things I wanted to break and then release a new version

**Comment by oli-obk** [comment]
Published 0.4.0

The only code that breaks is code exhaustively matching on the structs.

**Comment by oli-obk** [comment]
Maybe we should add a `Display` impl that reproduces the original `String`, so anyone who used the `workspace_member` before can just keep using it by calling `to_string()`

**Pull request merged**